lib/sass/script/value/map.rb

```diff
@@ -57,7 +57,7 @@ def to_sass(opts = {})
 
       to_sass = lambda do |value|
         if value.is_a?(Map) || (value.is_a?(List) && value.separator == :comma)
-          "(#{value.to_sass(opts)})"
+          "#{value.to_sass(opts)}"
```

Comment:
To match CSS, lists don't require parentheses unless it would be ambiguous otherwise, as here.